AudioKit/Common/Internals/Synchronization/AKTiming.swift

```diff
@@ -12,33 +12,93 @@
     /// Starts playback at a specific time.
     /// - Parameter audioTime: A time in the audio render context.
     ///
-    func play(at audioTime: AVAudioTime?)
+    @objc func start(at audioTime: AVAudioTime?)
 
     /// Stops playback immediately.
-    func stop()
+    @objc func stop()
 
-    /// Start playback immediately.
-    func play()
+    var isStarted: Bool { get }
```

Comment:
@ryanfrancesconi I removed play() from AKTiming it's just a convenience for play(at:nil) I'm sorry about isStarted, it had to be done.

Reply:
ok cool. I just committed an AKPlayer update, so I'm going to look at this now